### DISCUSSION TITLE
Pages user site not published

### DISCUSSION THREAD

**Discussion by signalshore** [community/community#23899] [Other Feature Feedback, Questions, & Ideas]
<p>I have a site at <a href="https://github.com/signalshore/signalshore.github.io/" rel="nofollow noopener">https://github.com/signalshore/signalshore.github.io/</a> that shows 404 when i try to visit it through <a href="http://signalshore.github.io" rel="nofollow noopener">signalshore.github.io</a></p>
<p>This has been happenning for almost 2 hours.</p>

**Comment by signalshore** [comment]
<p>Here are two screenshots</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/0a7ea1bfee04694fe80675712aa926e450d9b64e.png" data-download-href="/uploads/short-url/1uQ55OToAPb1SX1Ks0iWLoe5vum.png?dl=1" title="2018-01-03-053641_1366x768_scrot.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/25563888/181079885-587192db-424e-4ca8-a907-b3c4358b5367.png" alt="2018-01-03-053641_1366x768_scrot.png" data-base62-sha1="1uQ55OToAPb1SX1Ks0iWLoe5vum" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/0a7ea1bfee04694fe80675712aa926e450d9b64e_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/0a7ea1bfee04694fe80675712aa926e450d9b64e_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/0a7ea1bfee04694fe80675712aa926e450d9b64e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/0a7ea1bfee04694fe80675712aa926e450d9b64e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2018-01-03-053641_1366x768_scrot.png</span><span class="informations">1366×768 72.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/83de8937fa5bfacb4122979ad6903c94361eacc5.png" data-download-href="/uploads/short-url/iOzkrSfQpWVDBfu6sAFKEzcKa6V.png?dl=1" title="2018-01-03-053650_1366x768_scrot.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/25563888/181079886-8335e525-d374-40ec-9306-acaf77383abf.png" alt="2018-01-03-053650_1366x768_scrot.png" data-base62-sha1="iOzkrSfQpWVDBfu6sAFKEzcKa6V" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/83de8937fa5bfacb4122979ad6903c94361eacc5_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/83de8937fa5bfacb4122979ad6903c94361eacc5_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/8/83de8937fa5bfacb4122979ad6903c94361eacc5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/83de8937fa5bfacb4122979ad6903c94361eacc5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2018-01-03-053650_1366x768_scrot.png</span><span class="informations">1366×768 40.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/signalshore">@signalshore</a></p>
<p>I just took a look, and your site appears to be live currently. </p>
<p>If you’re still seeing a 404 error, it’s possible your browser is serving a cached version of the page from before it was successfully published. If you happen to be using Chrome as your browser, you can use the keyboard shortcut Control+Shift+R to fix this. Alternatively, you could try visiting your website in an incognito or private browsing window.</p>
<p>It’s also good to note that it can take up to ten minutes for a GitHub Pages build to complete (and for your changes to appear) after pushing a new commit to your repository. So sometimes, it takes a few before you’ll see your site as you expect it.</p>
<p>Hope this helps!</p>

**Reply by IbadDE** [reply]
YOU ARE A HERO BROTHER. <3

**Comment by tarun4all** [comment]
<p>same thing happening with me for hours and site is not live till now. i have cleared my cache too, tried in incognito as well still not working what to do </p>
<p><a href="https://github.com/tarun4all/trackme.git.io" rel="nofollow noopener">https://github.com/tarun4all/trackme.git.io</a></p>

**Comment by sis-c** [comment]
<p>You can’t have a subdomain that isn’t your username, and if you do a differently named repo, it will not automatically get set as a github page. Go to the repo settings and turn on github pages on the master branch. Then, your page should be at <a href="https://tarun4all.github.io/trackme.git.io/" rel="nofollow noopener">https://tarun4all.github.io/trackme.git.io/</a> because of how you named your repo.</p>

**Comment by derahn** [comment]
<p>Hello, im presently also having the same issue, it all started with deploying through m terminal, first got the</p>
<p>fatal: A branch named ‘gh-pages’ already exists.</p>
<p>Solved this by deleting the node_modules/.cache folder.</p>
<p>Then proceeded into re-running deployment and got another issue.</p>
<p>fatal: destination path ‘node_modules\gh-pages.cache\<a href="http://github.com" rel="nofollow noopener">github.com</a>!derahn!My-portfolio.git’ already exists and is<br>
not an empty directory.</p>
<p>Solved this by visiting my repo setting and dissabling the gh-pages section (setting source to none)</p>
<p>On doing this, everything seemed to work ok, with my terminal finally displaying the published solution.</p>
<p>But then its been about an hour and still getting the 404 error code.</p>
<p>Heres the link to the site: <a href="https://derahn.github.io/dotun_portfolio/" rel="nofollow noopener">https://derahn.github.io/dotun_portfolio/</a></p>
<p>Please advice accordingly, thanks</p>

**Comment by mistersuuns** [comment]
<p>Hello, newbie question: if I have a second repository in github, what do I call it then? (If the repository can only be called <a href="http://myusername.github.io" rel="nofollow noopener">myusername.github.io</a>, only that one github repository can be published)</p>

**Comment by mistersuuns** [comment]
<p>Ahhh, I reread your post and now understand how to resolve. Thanks!</p>

**Comment by Jameswilliams12** [comment]
<p>Same happend with my <a href="https://vloggingmaster.com" rel="nofollow noopener">site</a>, thank God I saw this post and my problem is solved</p>

**Comment by Dalexz4** [comment]
<p>Same Thing happened to my hosted site too. Page not found. How do I fix this?</p>

**Comment by noworry-b-still** [comment]
<p>hello,</p>
<p>   even i have am facing the samr problem…i have tried what u have mentioned in the post but still it’s showing page not found</p>
<p>can u help me in this regard please…</p>

**Comment by alexjhidalgo** [comment]
<p>I had this issue and the culprit turned out to be that I had created a repo just before with the same name and then deleted it. When I renamed the repo of interest the live page loaded up just fine.</p>

**Comment by belanger** [comment]
<p>I have the same problem that a lot of people are having, but don’t see how it is getting fixed.</p>
<p>I was hoping to use the address &lt;username&gt;.github,io,</p>
<p>I first tried creating the pages in a repository called /docs, but under settings -&gt; GitHub pages -&gt; Source,</p>
<p>the option to use docs as the source was greyed out.  Then I tried creating a repository</p>
<p>&lt;username&gt;.github.io, and when I do that, under settings -&gt; GitHub pages -&gt; Source, rather than</p>
<p>giving me a menu, it says that “Your GitHub Pages site is currently being built from the master branch”</p>
<p>and above that it says “Your site is ready to be published at <a href="http://belanger.github.io/" rel="nofollow noopener">http://&lt;username&gt;.github.io/</a>”</p>
<p>Sounds good, but when I click that link I get a 404 page.  I’m sure I’m missing something</p>
<p>(maybe “ready to be published” is different that “published” and I need to do something else),</p>
<p>but I don’t know what.</p>
<p>Any ideas?</p>

**Comment by belanger** [comment]
<p>Well, never mind.</p>
<p>After I changed the name to my repository, I had to make a commit before it would get published.  But not it’s up.</p>

**Comment by vipin-tech** [comment]
<p>My site is not published. <a href="http://www.vipin-tech.github.io." rel="nofollow noopener">www.vipin-tech.github.io. </a></p>
<p>When I login to github and navigate to settings page and go to github pages url, I am able to access the website.</p>
<p>But when I clear cookies and cache and type the url in browser, I am seeing 404 error.</p>
<p>Please can anyone help me with this?</p>

**Comment by roopamgarg** [comment]
<p>You can publish your other repositories as well using Github pages but the domain from which you can access it is &lt;your-username&gt;.github.io/&lt;your-repo-name&gt;</p>

**Comment by AmShubhangi** [comment]
<p>Still its 404 Bro!!</p>

**Comment by abacadeveloper** [comment]
<p>My site is still not available and I am experiencing the error 404! Anyone was able to see their websites live?</p>

**Comment by ManaalS** [comment]
<p>Did you figure out the issue? I’m having the same one :open_mouth:</p>

**Comment by VedantKarpe** [comment]
<p>I assume you were able to push the code successfully to the repo.</p>
<p>Make sure the following things are in place:</p>
<ol>
<li>Angular Cli Version should be 8+.</li>
<li>Repo should be public and not private - navigate to settings and change the repo to public.</li>
<li>Try the following command: ng deploy --repo=https://github.com/&lt;userID&gt;/&lt;repoName&gt; --base-href=https://&lt;userID&gt;.github.io/&lt;repoName&gt;/</li>
</ol>
<p>In the settings page, you’ll get the generated link.<br>
Check the network tab to see if the bundles are being loaded successfully, else clear the cache once and try : ctrl + shift + R.</p>

**Comment by Ashishsaha12** [comment]
My site is not published though live link is generated.
https://github.com/Ashishsaha12/Portfolio-Project_colorlib.git

**Comment by Naveenpandey27** [comment]
I am facing 404 error can anyone help me publishing mu website?

**Comment by Ruslanjan1987** [comment]
Hi how are you????please help me 404 page not found????
